skrub/_drop_column_if_null.py

```diff
@@ -25,8 +40,16 @@ def fit_transform(self, column, y=None):
         """
         del y
 
-        self.drop_ = sbd.is_all_null(column)
-
+        if self.threshold == 1.0:
+            self.drop_ = sbd.is_all_null(column)
+        elif self.threshold is None:
+            self.drop_ = False
+        else:
+            n_count = sum(sbd.is_null(column))
+            if n_count / len(column) > self.threshold:
```

Comment:
maybe this can be simplified to `self.drop_ = n_nulls / len(column) > self.threshold`? also we probably want to handle the len=0 corner case